src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.csproj

```diff
@@ -43,8 +43,11 @@
     <RunCodeAnalysis>false</RunCodeAnalysis>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.Management.Network, Version=19.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.7.0-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.KeyVault, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.KeyVault.2.4.3\lib\net452\Microsoft.Azure.Management.KeyVault.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.Network, Version=19.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Network.19.6.2-preview\lib\net452\Microsoft.Azure.Management.Network.dll</HintPath>
```

Comment:
@akshaysngupta the version of the Network SDK across all projects is 19.7.0-preview, is there any reason to downgrade it here?

Reply:
removed.